src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,127 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classes from 'classnames';
 import './FindMovie.scss';
 
+import { getMovie } from '../../api/api';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+export const FindMovie = ({ addMovie }) => {
+  const [movieTitle, updateMovieTitle] = useState('');
```

Comment:
function from useState always named as - "set" + name of variable. if you have movieTitle variable, function which changes it gonna be setMovieTitle

Reply:
change everywhere